encrypted-media-respec.html

```diff
@@ -3532,17 +3546,34 @@
             Applications MAY decode the contents of the ArrayBuffer to a JSON string using the <a def-id="interface-textdecoder"></a> [[!ENCODING]].
           </p>
 
-          <section id="clear-key-release-format-example" class="informative">
-            <h5>Example</h5>
-            <p>The following example is a license release for a session which contained two keys. (Line breaks are for readability only.)</p>
+          <section id="clear-key-release-format-example-1" class="informative">
+            <h5>Example message reflecting a <var>record of key usage</var></h5>
```

Comment:
I would reverse the examples to be consistent with the order above (simpler first).